content/en/docs/releases/kubeflow-1.7.md

```diff
@@ -0,0 +1,272 @@
++++
+title = "Kubeflow 1.7"
+description = "Information about the Kubeflow 1.7 release"
+weight = 98
```

Comment:
The weight needs to be lower than the 1.6 release page to allow the 1.7 release page to be displayed first

currently, https://deploy-preview-3476--competent-brattain-de2d6d.netlify.app/docs/releases/ shows 1.6 first